frontend/src/store/slices/systemConstants/useSystemConstants.ts

```diff
@@ -2,15 +2,28 @@ import * as actionTypes from 'constants/actionTypes';
 import { useApiSystemConstants } from 'hooks/pims-api/useApiSystemConstants';
 import React from 'react';
 import { hideLoading, showLoading } from 'react-redux-loading-bar';
-import { useAppDispatch } from 'store/hooks';
+import { useAppDispatch, useAppSelector } from 'store/hooks';
 
 import { IGenericNetworkAction } from '../network/interfaces';
 import { logError, logRequest, logSuccess } from '../network/networkSlice';
 import { ISystemConstant, storeSystemConstants } from '.';
 
+export enum SystemConstants {
+  FYEND = 'FYEND',
+  FYSTART = 'FYSTART',
+  GST = 'GST',
```

Comment:
Glad you added those ones. I think moving the enum to interfaces/ISystemConstant.ts  makes more sense, but its a nit.